renderer/asciidoctor.go

```diff
@@ -140,3 +141,15 @@ func (adr *AsciidoctorRenderer) RenderGVLink(gv types.GroupVersionDetails) strin
 func (adr *AsciidoctorRenderer) RenderAnchorID(id string) string {
 	return fmt.Sprintf("%s%s", asciidocAnchorPrefix, adr.SafeID(id))
 }
+
+func (adr *AsciidoctorRenderer) Escape(text string) string {
+	escaped := ""
+	for _, r := range text {
+		if r == '|' {
+			escaped += "\\|"
+		} else {
+			escaped += string(r)
+		}
+	}
+	return escaped
+}
```

Comment:
```suggestion
func (adr *AsciidoctorRenderer) RenderFieldDoc(text string) string {
	// escape the pipe character, which has special meaning for asciidoc as a way to format tables,
	// so that including | in a comment does not result in wonky tables
	return strings.ReplaceAll(text, "|", "\\|")
 }
```

Reply:
Sorry for a last change, looking at the code with fresh eyes, I think it would be more consistent with the existing to name the function `RenderFieldDoc` and good to add a little comment about why we are escaping the pipe character. We can also use `strings.ReplaceAll`.